cli/onionshare_cli/resources/static/js/chat.js

```diff
@@ -133,6 +133,15 @@ var checkUsernameExists = function (username) {
   return false;
 }
 
+var checkUsernameLength = function (username) {
```

Comment:
Maybe change the function name to `checkUsernameTooLong`? from the function name it's not very clear right now what it is doing.